### PR TITLE
Add package prelink removed

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:
* Add new rule package_prelink_removed
* Add package_prelink_removed to RHEL 7 CIS profile

#### Rationale:
Increase coverage for RHEL 7 CIS.

### CODE REVIEW

File: linux_os/guide/system/software/integrity/package_prelink_removed/ansible/shared.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = disable
+# complexity = medium
+# disruption = low
+
+- name: Restore Prelinked Binaries
+  ansible.builtin.shell:
+    cmd: prelink -ua
```
Comment (jan-cerny) [inline linux_os/guide/system/software/integrity/package_prelink_removed/ansible/shared.yml:9]: I think that this should be limited only to the situation when prelink is installed.

**Review by jan-cerny — CHANGES_REQUESTED** [review]
The automatus CI fail is caused by the fact that this rule has `prodtype: rhel7` therefore the rule is RHEL 7 only and the rule doesn't exist on products where automatus runs.

File: linux_os/guide/system/software/integrity/package_prelink_removed/ansible/shared.yml (hunk 1 above)
Comment (Mab879) [reply linux_os/guide/system/software/integrity/package_prelink_removed/ansible/shared.yml:15]: I have applied this suggestion to the bash remediation as well.

**Review by jan-cerny — APPROVED** [review]
The automatus CI fail is caused by the fact that this rule has prodtype: rhel7 therefore the rule is RHEL 7 only and the rule doesn't exist on products where automatus runs.

```
[jcerny@thinkpad scap-security-guide{pr/10062}]$ python3 tests/automatus.py rule  --libvirt qemu:///system ssgts_rhel7 package_prelink_removed
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/scap-security-guide/logs/rule-custom-2023-01-17-0935/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_package_prelink_removed
INFO - Script package-removed.pass.sh using profile (all) OK
INFO - Script package-installed-removed.pass.sh using profile (all) OK
INFO - Script package-installed.fail.sh using profile (all) OK
[jcerny@thinkpad scap-security-guide{pr/10062}]$ python3 tests/automatus.py rule  --libvirt qemu:///system ssgts_rhel7 --remediate-using ansible package_prelink_removed
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/scap-security-guide/logs/rule-custom-2023-01-17-0938/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_package_prelink_removed
INFO - Script package-installed.fail.sh using profile (all) OK
INFO - Script package-installed-removed.pass.sh using profile (all) OK
INFO - Script package-removed.pass.sh using profile (all) OK

```

**Pull request merged**